pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -841,6 +841,7 @@ func (m *kubeGenericRuntimeManager) removeContainerLog(containerID string) error
 	}
 	labeledInfo := getContainerInfoFromLabels(status.Labels)
 	annotatedInfo := getContainerInfoFromAnnotations(status.Annotations)
+	// TODO(random-liu): Add container log directory and cleanup directory.
```

Comment:
If this is for a future PR, please open an issue and include the issue number in the comment, e.g. `// TODO(#issue): issue-title`.